redash/settings/__init__.py

```diff
@@ -16,6 +17,8 @@
 )
 from redash.settings.organization import DATE_FORMAT, TIME_FORMAT  # noqa
 
+load_dotenv()
+
```

Comment:
Thanks a lot for your feedback on this @berinhard . I had to rebase my source branch here to be updated with metr-main because this update regarding the usage of `dotenv` has been already merged there in an earlier PR that you can find [here](https://github.com/metr-systems/redash/pull/2/files) . As far as i remember , the project was not uploading the needed en variable before adding `load_dotenv()`